packages/storefront/src/components/Navigation.vue

```diff
@@ -62,6 +62,10 @@
       return `/${params.map((x) => paramCase(x)).join('/')}`;
     }
 
+    public isExtendedActive(page?: string) {
```

Comment:
This should always be defined since it is rendered in L22. Also if it wasn't defined, then `paramCase` would fail.
```suggestion
    public isExtendedActive(page: string) {
```